indico/modules/events/contributions/util.py

```diff
@@ -222,33 +223,50 @@ def import_contributions_from_csv(event, f):
         if not title:
             raise UserValueError(_("Row {}: contribution title is required").format(num_row))
 
+        contrib_data.append({
+            'start_dt': parsed_start_dt,
+            'duration': parsed_duration or timedelta(minutes=20),
+            'title': to_unicode(title),
+            'speaker': {
+                'first_name': to_unicode(first_name),
+                'last_name': to_unicode(last_name),
+                'affiliation': to_unicode(affiliation),
+                'email': to_unicode(email)
```

Comment:
What about rejecting imports containing obviously invalid emails? I've seen at least one failure related to someone importing a CSV containing a row with headers.